backend/src/test/java/de/silvia/backend/security/services/UserServiceTest.java

```diff
@@ -18,7 +20,9 @@
     @Test
     public void loadUserByUsername()  {
         assertNotNull(userRepo);
-        User mockedUser = new User("Julius", "123456");
+        User mockedUser = User.newUser("Julius", "jjj@ddd.cd", "Schmitz",
+                "userX", "kskskk", List.of(
+                        new SimpleGrantedAuthority("API_READWRITE")));
         when(userRepo.findByUsername("Julius")).thenReturn(java.util.Optional.of(mockedUser));
         UserService t  = new UserService(userRepo);
         User check = t.loadUserByUsername("Julius");
```

Comment:
a more common naming convention is to use actual vs expected, check could imply that this variable is a check result, which it is not

Reply:
Good hint, thanks!